v3/plugins/ms-vscode/protobuf/0.4.2/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: ms-vscode
+name: protobuf
+version: 0.4.2
+type: VS Code extension
+displayName: Protobuf
+title: Google's protobuf .proto file support
+description: This extension adds rich language support for the proto3 syntax
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
```

Comment:
https://github.com/zxh0/vscode-proto3/blob/master/images/vscode_extension_icon.png could be reused as an icon. @vitaliy-guliy  WDYT?

Reply:
I think this is more accurate since the extension also supports proto2
![image](https://user-images.githubusercontent.com/407397/91318865-157d5c00-e792-11ea-8604-c4284ac01db2.png)

Reply:
![image](https://user-images.githubusercontent.com/407397/91318919-229a4b00-e792-11ea-8bb1-7b084dd2adf6.png)

Reply:
![image](https://user-images.githubusercontent.com/407397/91318983-39d93880-e792-11ea-8e1a-b223d398139f.png)

Reply:
Ideally it would be an icon from the (or a) repository somewhere.

Reply:
can it be included in plugin registry in some subfolder? I don't see any "officially" maintained image neither in protocol buffers repo nor extension's repo)

Reply:
As for me, we reuse vscode-extension, so, I don't see any problems to use this icon https://github.com/zxh0/vscode-proto3/blob/master/images/vscode_extension_icon.png from the extension repository.

Reply:
The extension in this PR supports proto3 right? In that case I think we can just use ^